build.gradle

```diff
@@ -132,9 +131,30 @@ task runtimeJar(type: Jar) {
     }
 }
 
+task engineJar(type: Jar) {
+    dependsOn compileJava
+    archiveBaseName = 'rhino-engine'
+    from (sourceSets.main.output) {
+        include 'org/mozilla/javascript/engine/**'
+        include 'META-INF/services/**'
+    }
+    manifest {
+        attributes(
+                "Manifest-Version": "1.0",
+                "Implementation-Version": project.version,
+                "Implementation-Title":  "Mozilla Rhino ScriptEngine",
+                "Implementation-Vendor": "Mozilla Foundation",
+                "Implementation-URL": "http://www.mozilla.org/rhino",
+                "Built-Date": new Date().format("yyyy-MM-dd"),
```

Comment:
Adding a timestamp to the manifest is a bad idea, it makes the jar non reproducible.